ts/main.ts

```diff
@@ -287,6 +287,18 @@ async function fetchData(usrInput: string): Promise<void> {
       monsterImgContainer.appendChild(monsterImg);
       $selectedSq.appendChild(monsterImgContainer);
       $selectedSq.classList.add('bg-green-500', 'correct');
+      $selectedSq.classList.remove('hover:bg-yellow-100');
+      numCorrect++;
+      guesses++;
+      completedSquares++;
+      setStatsText();
+      clearSelectedSqResetModal($selectedSq);
+    } else {
+      guesses++;
+      completedSquares++;
```

Comment:
Should we be setting the `completedSquares` here in the `else` if the user get the question wrong?

Reply:
With how I have it set up right now, if a user gets a wrong answer the square turns red and they can no longer try to solve that square, so if they get an answer wrong, another square is 'completed' cause it's no longer in play. However, not sure if I want to keep it this way or let user's keep guessing until all the squares are filled